core.ts

```diff
@@ -17,8 +17,6 @@ class ChartSvgCanvas extends SvgCanvas {
 
 /** Underlying ChartJS defaults which can be modified. */
 export const defaults: ChartJs.Defaults = ChartJs.defaults;
-/** Underlying ChartJS plugins. */
-export const plugins = ChartJs.plugins;
```

Comment:
Removed this because this isn't available in typing since v4.2.0. I couldn't find the usage of this in the examples & user repos. I think we don't need to re-export this.